pulsar-client-api/src/main/java/org/apache/pulsar/client/api/schema/GenericSchema.java

```diff
@@ -40,17 +39,4 @@
      * @return generic record builder
      */
     GenericRecordBuilder newRecordBuilder();
-
-
-    static GenericSchema of(SchemaInfo schemaInfo) {
```

Comment:
You can't just remove methods from a public API because it will cause code incompatibility issues.

Reply:
OK, I will recover it